### PR TITLE
`@CanIgnoreReturnValue` in `Builder`s

### PR DESCRIPTION
**Pull request by armiol**
This PR updates the `Builder` API with `@CanIgnoreReturnValue` annotations where they are needed.

The library version is set to `2.0.0-SNAPSHOT.66`.

### CODE REVIEW

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt [hunk 1]
```diff
@@ -237,18 +245,29 @@
...
     ) {
+        val inputs = composeInputs(tasks, allowInternalJavadoc)
         tasks.register(taskName, Copy::class.java) {
             doLast {
-                if (allowInternalJavadoc) {
-                    from(tasks.javadocTask(InternalJavadocFilter.taskName))
-                } else {
-                    from(tasks.javadocTask(JavadocTask.name))
-                }
+                from(*inputs.toTypedArray())
```
[the remark this answers is not in the document]
Comment (armiol) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt:251]: @sonatype-lift ignore

File: tests/buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt [hunk 2]
```diff
@@ -237,18 +245,29 @@
...
     ) {
+        val inputs = composeInputs(tasks, allowInternalJavadoc)
         tasks.register(taskName, Copy::class.java) {
             doLast {
-                if (allowInternalJavadoc) {
-                    from(tasks.javadocTask(InternalJavadocFilter.taskName))
-                } else {
-                    from(tasks.javadocTask(JavadocTask.name))
-                }
+                from(*inputs.toTypedArray())
```
[the remark this answers is not in the document]
Comment (armiol) [reply tests/buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/UpdateGitHubPages.kt:251]: @sonatype-lift ignore

**Comment by armiol** [comment]
@alexander-yevsyukov @dmdashenkov PTAL.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM

**Pull request merged**